common/services/move.test.js

```diff
@@ -74,6 +74,14 @@ describe('Move Service', function () {
         'profile.person_escort_record.responses.nomis_mappings',
         'profile.person_escort_record.flags',
         'profile.person_escort_record.prefill_source',
+        'profile.youth_risk_assessment',
```

Comment:
Is it possible to order the includes alphabetically? I've just done that in my branch and I think it will make it easier to merge etc